components/DesaDigital/LevelOne/index.vue

```diff
@@ -42,7 +48,46 @@ export default {
         totalRows: 0,
         itemsPerPage: 5,
         itemsPerPageOptions: [],
-        disabled: true
+        disabled: false
+      },
+      query: {
+        name: null,
+        per_page: 5,
+        current_page: 1,
+        level: 1,
+        is_active: true
+      }
+    }
+  },
+  async fetch () {
+    try {
+      const response = await this.$axios.get('/villages/list-with-location', { params: this.query })
+      const { data, meta } = response.data
+      this.data = data || []
+      this.pagination.currentPage = meta?.current_page || 1
+      this.pagination.totalRows = meta?.total || 0
+      this.pagination.itemsPerPage = meta?.per_page || this.query.per_page
+    } catch (error) {
+      this.pagination.disabled = true
+    }
```

Comment:
Ask:

Hello @yoslie, I wondering why to put the `error` as a parameter in the catch but It's never used? 

#cmiiw

Reply:
open discuss:
Just to make sure it's catch an error lol,
actually I don't know is if it's must be used or not. But different from the declaring variable inside the function, eslint didn't see it as unused variable so I dont remove it just as a sign that it catch an error. #cmiiw 
But I can remove it if u want 😄

Reply:
During the param in catch block is never used and we have statements to handle any exceptions. So, we can remove that. wdyt? 😃

Reply:
sure changes [here](https://github.com/jabardigitalservice/dedi-cms/pull/88/commits/e317d7ef6f2b7f86dba669ce42680f368e9af732) thankyou 🙏